### ISSUE TITLE
x/website: dead link for change log

### ISSUE THREAD

**Issue by rittneje** [golang/go#44068]
The [release notes page](https://golang.org/doc/devel/release.html) has a link to a change log, but the link is dead.

https://github.com/golang/website/blob/4c40783eaee9044e2e3ec516fe05a7e5c50e1fb0/content/static/doc/devel/release.html#L6-L7

**Comment by ALTree** [comment]
Points to golang.org/change which works for me. What do you mean by dead? Are you getting a 404?

**Comment by rittneje** [comment]
I am getting redirected to https://go.googlesource.com/go. Is that commit list supposed to be the change log?

**Comment by ALTree** [comment]
Yes, that's correct. https://go.googlesource.com/go is the canonical path of the project source code (the github repo is just a read-only mirror). So the link is essentially saying "look at the repository history (hosted here) for an exhaustive list of changes committed to the codebase".

The wording around is maybe a little awkward, but the link points where it should.
